introduction/java_loops_II

```diff
@@ -0,0 +1,25 @@
+import java.util.*;
+import java.io.*;
+import java.lang.Math;
+
+class Solution{
+  public static void main(String []argh) {
```

Comment:
Longer implementations place in a separate methods which can be called from main. main always should be very short.